docker/dagmc-viz.dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM  ubuntu:18.04
```

Comment:
Maybe start from a existing docker image ? like the one we use for PyNE CI?

Reply:
You could start from the svalinn/pymoab-py2-18.04 and then you just need to install Visit (and maybe a few extra apt packages).